engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -1068,12 +1071,48 @@ private void finishApplication(FinalApplicationStatus finalStatus) throws YarnEx
    * Check for containers that were allocated in a previous attempt.
    * If the containers are still alive, wait for them to check in via heartbeat.
    */
-  private void previouslyAllocatedContainers(List<Container> containers)
+  private void previouslyAllocatedContainers(List<Container> containersListByYarn, Collection<ContainerId> releasedContainers)
   {
-    for (Container container : containers) {
-      this.allocatedContainers.put(container.getId().toString(), new AllocatedContainer(container));
-      //check the status
-      nmClient.getContainerStatusAsync(container.getId(), container.getNodeId());
+    LOG.info("Finding information about the previously allocated containers.");
+
+    if (containersListByYarn.size() != 0) {
+
+      LOG.info("YARN has the list of {} running containers.", containersListByYarn.size());
```

Comment:
Why two log entries? IMO, none of them are necessary at least at info level. Why not to log all containers in debug?